problems/arrayMethods.js

```diff
@@ -145,7 +221,14 @@ const numberTimesIdx = () => {};
  * @returns {boolean} Are all numbers positive, even, and not ending in 0.
  */
 
-const arePositiveEvenAndNonZeroEnding = () => {};
+const arePositiveEvenAndNonZeroEnding = (nums) => {
+  return nums.every((el) => {
+    if (el > 0 && el % 10 !== 0){
```

Comment:
It's interesting that you didn't check for a number being even, because if it ends in 0 it will definitely be even!